assets/src/stories-editor/blocks/amp-story-cta/edit.js

```diff
@@ -8,128 +8,97 @@ import PropTypes from 'prop-types';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
-import {
-	Dashicon,
-	IconButton,
-} from '@wordpress/components';
-import {
-	URLInput,
-	RichText,
-} from '@wordpress/block-editor';
+import { useState, useEffect } from '@wordpress/element';
+import { Dashicon, IconButton } from '@wordpress/components';
```

Comment:
Why were these changed a single line?

Reply:
Didn't really make much sense to have these on separate lines as these are only two imports. So while I was it, I tidied up the imports here a bit.

I'm planning on configuring ESLint to enforce these kind of things if possible.